index.html

```diff
@@ -0,0 +1,339 @@
+        </ul>
+      </div>
+    </section>
+    <header class="main-header">
+      <nav>
+        <ul class="header-menu">
+          <li><a href="#" class="info-text">Melkamu</a></li>
+          <li>
+            <a href="#" id="hamburger-btn" class="nav__button"
+              ><i
+                ><img
+                  src="Asset/Icon - Menu.png"
+                  alt="hamburger-icon-menu"
+                  class="info-image"
+                  id="hamburger-btnicon" /></i
+            ></a>
+          </li>
+          <li class="desktop-menu-container">
+            <ul class="menu-lists">
+              <li class="item"><a href="#portfolio">Portfolio</a></li>
+              <li class="item"><a href="#about">About</a></li>
+              <li class="item"><a href="#contact">Contact</a></li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-content">
+      <section class="headlines">
+        <div class="container">
+          <div class="about" id="about-info">
+            <h1 class="intro">I’m Melkamu.<br />Glad to see you!</h1>
+            <p>
+              I'm a software developer! I can help you build a product, feature,
+              or website. Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hesitate to contact me.
+            </p>
+          </div>
+          <h3 class="connect">LET’S CONNECT</h3>
+          <ul class="social">
+            <li>
+              <a href=""><img src="Asset/vector1git.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/LinkidIn.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/angellist.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/Twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/medium.png" alt="" /></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="card" id="works">
+        <div class="card-box">
+          <img
+            src="Asset/Snapshoot Portfolio.png"
+            alt="Tonic Company"
+            class="tonic-image"
+          />
+        </div>
+        <div class="card-box">
+          <h1 class="TonicHeader">Tonic</h1>
+          <div class="HeaderTitle">
+            <h2 class="tonic-title">CANOPY</h2>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">Back End Dev</h3>
+            <img src="Asset/Counter.png" class="counter" />
+            <h3 class="TonicEnd">2015</h3>
+          </div>
+          <p class="Content">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill-box">
+            <li>html</li>
```

Comment:
- [ ] Kindly reduce the padding in the button item, and increase the border-radius of the cards.

- Figma design
<img width="309" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/f9a36276-dce2-4254-9557-5c7f38b33248">

- Your design
<img width="425" alt="image" src="https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/41163571/fc232cee-4d2d-4c39-b0f0-c69bcd4c6a7d">